include/fizzy/fizzy.h

```diff
@@ -10,6 +10,13 @@
 #include <stddef.h>
 #include <stdint.h>
 
+/// Safe way of marking a function with `noexcept` C++ specifier.
+#ifdef __cplusplus
+#define FIZZY_NOEXCEPT noexcept
```

Comment:
I think `NOEXCEPT` would be fine provided it is wrapped with `#ifndef NOEXCEPT`.

Reply:
Are you keen on this? This is a public header, so who knows what projects could define `NOEXCEPT` as. This feels a bit safer -- less like to clash.